docs/contents/cloud-asset.json

```diff
@@ -0,0 +1,81 @@
+{
+    "title": "Asset",
+    "defaultService": "asset\/readme",
+    "services": [
+        {
+            "title": "Overview",
+            "type": "asset\/readme"
+        },
+        {
+            "title": "AssetService",
+            "type": "asset\/metadata\/v1beta1\/assetservice",
+            "nav": [
+                {
+                    "title": "Assets",
+                    "type": "asset\/metadata\/v1beta1\/assets"
+                }
+            ],
+            "patterns": [
+                "asset\/metadata\/v1beta1\/\\w{1,}"
+            ]
+        },
+        {
+            "title": "AssetServiceClient",
+            "type": "asset\/v1beta1\/assetserviceclient",
+            "nav": [
+                {
+                    "title": "Asset",
```

Comment:
To this point, we've not included the proto classes in the table of contents. Is this something you think we should reconsider going forward?

Reply:
No, it's just because I still don't fully understand the `component` subcommand's questions.

Reply:
They're removed (in the next commit)